packages/video_player/video_player_platform_interface/lib/messages.dart

```diff
@@ -132,17 +132,20 @@ class PositionMessage {
 
 class MixWithOthersMessage {
   bool? mixWithOthers;
+  bool? ambient;
```

Comment:
The changes to this file can also be reverted, and instead applied to the messages.dart in video_player_avfoundation.